### ISSUE TITLE
Checkpoint file cannot be found error on Kubernetes cluster

### ISSUE THREAD

**Issue by shouhong** [tensorflow/ecosystem#28]
I followed the instructions from _**https://github.com/tensorflow/ecosystem/tree/master/kubernetes**_ to run mnist sample on kubernetes with 2 worker and 2 ps. Then I got below errors. Do you have any idea what's the possible reason? Thanks!

1. Error on ps-1: **Not found: /train_dir/model.ckpt-0_temp_5fcd97c881a7428db8eded829b964618/part-00000-of-00002.index**

2. Error on worker-0: **NotFoundError (see above for traceback): /train_dir/model.ckpt-0_temp_5fcd97c881a7428db8eded829b964618/part-00000-of-00002.index
	 [[Node: save/MergeV2Checkpoints = MergeV2Checkpoints[delete_old_dirs=true, _device="/job:ps/replica:0/task:1/cpu:0"](save/MergeV2Checkpoints/checkpoint_prefixes, _recv_save/Const_0_S81)]]**

The **_mnist.py_** used is from the _**https://github.com/tensorflow/ecosystem/tree/master/docker**_

**Comment by drinktee** [comment]
I have the same problem.

**Comment by jhseu** [comment]
Are you setting the train_dir to a local directory? It must be a directory visible to all workers.

**Comment by drinktee** [comment]
> Are you setting the train_dir to a local directory? It must be a directory visible to all workers.

Yes,I set the train dir to a local directory.Let me try it again...

**Comment by aligokalppeker** [comment]
I have the same problem. But i use "google colab" and google drive ocaml fuse not in a distributed fashion. How can i solve this problem ? It occurs on some models. Maybe it depends on the model structure.

Is there any way to change the temporary directory for checkpoint saver ?
